uavcan/equipment/ahrs/1002.PositionAttitude.uavcan

```diff
@@ -0,0 +1,40 @@
+#
+# Inertial data and orientation in body frame with fused location
+#
+
+uavcan.Timestamp timestamp
+
+float64 longitude                   # angular degree
+float64 latitude                    # angular degree
+float32 height_ellipsoid            # meter
+float32 height_msl                  # meter
+float32 height_baro                 # meter
+float32 ground_distance             # meter (with radar altimeter or LIDAR)
```

Comment:
Yes, in aviation baro is used a lot but it also changes without "real" connection to any physical altitude. For example when plane is in vicinity of airplane zero baro is set from tower, when in free flight, all baro's zeros on all planes are set to same value.
But this value is really needed to be reported.
For height_agl this one is strictly based on ground and follow terrain but is very good for photo missions to store.